### PR TITLE
SCRUM-2823 fix evidence code Column of disease annotation table on gene page

### PR DESCRIPTION
**Pull request by cmpich**
…aryAnnotation object.

### CODE REVIEW

**Review by markquintontulloch — CHANGES_REQUESTED** [review]
This doesn't make sense to me.  You are getting the URL template from the crossReference dataProvider, but then applying the DOID from the object curie to that template.  If the cross reference on the data provider is not linking to the desired page then that is a data issue.

**Comment by cmpich** [comment]
Removed the commit that changed the crossReference curie setting. That issue will fixed with a data migration script.

File: agr_api/src/main/java/org/alliancegenome/api/translators/tdf/DiseaseAnnotationToTdfTranslator.java [hunk 1]
```diff
@@ -325,6 +325,19 @@ private static StringJoiner getStringJoiner(GeneDiseaseAnnotationDocument annota
 		return evidenceJoiner;
 	}
 
+	private static StringJoiner getStringJoiner(org.alliancegenome.curation_api.model.entities.DiseaseAnnotation annotation, Function<org.alliancegenome.curation_api.model.entities.ontology.ECOTerm, String> function) {
```
Comment (oblodgett) [inline agr_api/src/main/java/org/alliancegenome/api/translators/tdf/DiseaseAnnotationToTdfTranslator.java:328]: @cmpich do we need the full package path to these classes?

**Review by markquintontulloch — APPROVED** [review]

File: agr_api/src/main/java/org/alliancegenome/api/translators/tdf/DiseaseAnnotationToTdfTranslator.java (hunk 1 above)
Comment (cmpich) [reply agr_api/src/main/java/org/alliancegenome/api/translators/tdf/DiseaseAnnotationToTdfTranslator.java:328]: Unfortunately, yes. We have a DiseaseAnnotation class in the agr_java_software repo.

**Pull request merged**